### ISSUE TITLE
A11y_Screen reader not announcing name of combo box it announcing only value of combo box.

### ISSUE THREAD

**Issue by ShaileshChudawkar** [microsoft/azuredatastudio#24218]
### Github tags
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #DSCT1.0.0; #FTP; #JAWS; #WCAG1.3.1; #ScreenReader; #Win10; #Win32; #NVDA:
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Actual Result:
Screen reader not announcing name of combo box it announcing only value of combo box.

**Similar issue observed with JAWS also.**
**Similar issue observed in:**

**1.  for 'File on, Extension and feedback ' control also.
2. 'Conversion options_Provide a name' Scenario also.**

### Expected Result:
Screen reader should announce name of combo box and value containing it also.
### Environment Details:
Application Name: Database Schema Conversion Toolkit Azure data studio
Version: 1.43.0 (System user)
Windows Version: Win11 Pro 21H2 
OS Build:22000.1817
ScreenReader: JAWS 2023.2303.144 /NVDA 2023.1
### Repro Steps:

1. Login to vm with appropriate credentials
2. Launch Azure data studio.
3. Tab to 'Manage Connection' and press enter
4. Tab to the dash board for Oracle extension
5. Tab to 'Feedback'
6. Turn on screen reader and observe the issue.
7. User Impact:
8. Screen reader user will not able to get complete information regarding combo box present in page.
### User Impact:
Screen reader user will not able to get complete information regarding combo box present in page.
### Attachments:
[2389897_With NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/1d46b191-036e-4e54-99b1-d845f3a098d7)
[2389897_With JAWS.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/bebaa99a-a198-40aa-80da-8aadc5a2309a)

**Comment by kisantia** [comment]
This issue is upstream. A tracking issue has been created in vscode (referenced above).

**Comment by cpichuka** [comment]
@kisantia , any ETA on this?

**Comment by kisantia** [comment]
@ShaileshChudawkar can this issue please be downgraded to sev3 or closed since an upstream issue has been created in vscode and there is no ETA for this to be fixed?

**Comment by kisantia** [comment]
@ShaileshChudawkar this has been fixed. Please try the[ latest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch) to verify the fix.

**Comment by PadminiGariki** [comment]
Verified the issue below environment.
Version: 1.48.0-insider (user setup)
Commit: 7bdf5c994bcc25aaeb60584dcaa24f83e6fde7fd
Date: 2024-01-29T22:20:11.186Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.23620

Issue is fixed. Hence the bug is closed.
#Regressed:02-01-24; #Closed;
For reference attachment is added.

[Screen Recording (35).webm](https://github.com/microsoft/azuredatastudio/assets/142479541/ac025314-4075-473b-8113-68dabe429110)

[24218_JAWS.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/36c345e2-d5f1-434f-8590-c0ff2f47a58c)
